src/main/java/org/folio/consortia/domain/entity/PermissionUser.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.consortia.domain.entity;
```

Comment:
@SerhiiNosko should it be dto instead of entity?

Reply:
yes correct, it should be in dto package

Reply:
fixed

Reply:
@SinghAdes I believe just dto not entity.dto? but better to confirm with @SerhiiNosko I'm not an expert :)

Reply:
yes, we don't need to include dto into entity package, basically lets move them to domain package

Reply:
and basically we are declaring all dtos in yaml, did you think about this approach?

Reply:
this dto is not used in request/response, probably it can be defined in java

Reply:
okay, I am to remain as is, just move to domain package

Reply:
fixed